ReportSimulationOutput/measure.rb

```diff
@@ -682,21 +682,21 @@ def check_for_errors(runner, outputs)
     end
 
     # Check sum of electricity produced end use outputs match total
-    sum_elec_produced = -1 * @end_uses.select { |k, eu| eu.is_negative }.map { |k, eu| eu.annual_output.to_f }.sum(0.0)
+    sum_elec_produced = -1 * @end_uses.select { |k, eu| eu.is_negative && k[1] != EUT::Battery }.map { |k, eu| eu.annual_output.to_f }.sum(0.0)
     total_elec_produced = outputs[:total_elec_produced]
-    if (sum_elec_produced - total_elec_produced).abs > 0.1
+    if (sum_elec_produced - total_elec_produced).abs > 0.05
```

Comment:
Why did you make these tolerances tighter?

Reply:
Because the battery energy was so low, it wasn't triggering this even though I was accounting correctly.

Reply:
Change this back since we have changes to initial SOC.